torch/testing/_internal/common_utils.py

```diff
@@ -734,6 +734,7 @@ def addSkip(self, test, reason):
                                     '--reruns=2', '-rfEX', f'--junit-xml-reruns={pytest_report_path}'])
             del os.environ["USING_PYTEST"]
             sanitize_pytest_xml(f'{pytest_report_path}')
+            print("Skip info is located in the xml test reports, please either go to s3 or the hud to download them")
```

Comment:
May be adding the name of the bucket and the oath here to help people navigate

Reply:
i think this would require adding a bunch of env vars, so im not gonna do that, it does say later in the log which s3 bucket and path they go to though